lua/terrortown/lang/en.lua

```diff
@@ -2174,3 +2174,6 @@ L.header_weapon_settings = "Weapon Settings"
 
 -- 2024-01-24
 L.grenade_fuse = "FUSE"
+
+-- 2024-02-04
+L.distance_meter = "{distance}m"
```

Comment:
I think "m" is a pretty universal standard globally and does not need to be translated

Reply:
I'm not sure it is m in chinese etc

Reply:
It is the official SI unit symbol there is nothing to translate